src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -428,16 +466,39 @@ private void updateTabBarVisible() {
      * Returns the currently viewed LibraryTab.
      */
     public LibraryTab getCurrentLibraryTab() {
```

Comment:
i think, this sshould be en Optional - don't know, how much effort this is.

Reply:
This is way beyond the scope of this pr. I checked the changes made, this is a bottomless pit.